src/main/java/com/grimbo/chipped/block/ChippedUniqueLantern.java

```diff
@@ -38,22 +39,24 @@ public VoxelShape getShape(BlockState blockState, BlockGetter blockGetter, Block
         return getShape(blockState);
     }
 
-    public VoxelShape getShape(BlockState state) {
+    public VoxelShape getShape(@NotNull BlockState state) {
         Direction direction = state.getValue(FACING);
         if (direction == Direction.EAST || direction == Direction.WEST) {
-            return toEast;
+            return this.toEast;
```

Comment:
Is there a reason for adding the explicit `this.` here?
If not, remove them. They're not necessary and don't really add anything other than bloat.

Reply:
it's pretty obvious when something is or isn't static by context, especially when things are immutable, so there isn't a need for that

Reply:
i'd rather keep it because i think it improves code readability, but you really want i can remove it